projects/ngx-maplibre-gl/src/lib/control/fullscreen-control.directive.ts

```diff
@@ -1,48 +1,49 @@
-import {
-  AfterContentInit,
-  Directive,
-  Host,
-  HostListener,
-  Input,
-} from '@angular/core';
+import { Directive, afterNextRender, inject, input } from '@angular/core';
 import { FullscreenControl } from 'maplibre-gl';
 import { MapService } from '../map/map.service';
 import { ControlComponent } from './control.component';
 
 /**
  * `mglFullscreen` - a fullscreen control directive
- * 
+ *
  * @category Directives
  */
 @Directive({
   selector: '[mglFullscreen]',
   standalone: true,
+  host: {
```

Comment:
Is there a better way to do this?

Reply:
I saw this in the new Angular doc:

https://angular.dev/guide/components/host-elements

![image](https://github.com/maplibre/ngx-maplibre-gl/assets/31312682/11a9f7f7-21ad-4d43-aab2-5af329074e28)

Reply:
Maybe we can remove it completely? See my comment below.